packages/functions/src/jobs/CronJobs.ts

```diff
@@ -0,0 +1,69 @@
+import { AppointmentStatus } from "common/src/types";
+import * as functions from "firebase-functions";
+import { getAppointmentsByStatus } from "../dal/AppointmentDataAccessLayer";
+import { getDonors } from "../dal/DonorDataAccessLayer";
+import { getAppointmentNotificationData } from "../notifications/AppointmentNotificationData";
+import {
+  NotificationToDonor,
+  sendEmailToDonor,
+} from "../notifications/NotificationSender";
+
+export const ConfirmationReminderOnSameDay = functions.pubsub
```

Comment:
I know it is not going to be easy, but please find a way to write tests for this job.

Let me know if you need help with it